manifests/entandoplugins.entando.org.crd.yaml

```diff
@@ -86,14 +80,7 @@ spec:
                 tlsSecretName:
                   description: >
                     The name of a standard Kubernetes TLS Secret, assumed to be in the same Namespace as the EntandoPlugin
-                    being created. This Secret is expected to have a private key, and a certificate for the hostname that
-                    the EntandoPlugin microservice will be exposed on. Please refer to the 'ingressHostname' property for more
-                    information on how the hostname is determined. If absent, the Entando Operator will evaluate the
-                    value of the ENTANDO_PATH_TO_TLS_KEYPAIR which is expected to contain two files: tls.key and tls.crt.
-                    If a keypair is found in the folder specified, it will revert to the keypair found. Alternatively,
-                    the Entando Operator will evaluate the value of the ENTANDO_USE_AUTO_CERT_GENERATION. If set to 'true'
-                    the Entando Operator will assume that the cluster has been configured with a valid CA and leave it to
-                    the Ingress controller in use to generate its own certificates.
+                    being created. See here: TODO for details and instructions on creating a TLS secret for your Entando applications.
```

Comment:
TODO?  You could point to the tutorial on the operator since I have the details on creating secrets there.

Reply:
This should be the URL to the tutorial page - https://dev.entando.org/v6.3/tutorials/devops/installation/open-shift/openshift-install-by-operator-hub.html